symengine/expression.h

```diff
@@ -173,11 +173,6 @@ inline Expression expand(const Expression &arg)
     return expand(arg.get_basic());
 }
 
-inline bool unified_eq(const Expression &a, const Expression &b)
```

Comment:
Are you sure? Can you send an error log? If there is an error, we should fix that.